business-registry-dashboard/app/locales/fr-CA.ts

```diff
@@ -213,7 +214,7 @@ export default {
         },
         400: {
           title: "Erreur Lors De L'Ajout De La Demande De Nom",
-          description: "Nous n'avons pas pu trouver de demande de nom associée au numéro de téléphone ou à l'adresse courriel que vous avez saisie. Veuillez réessayer."
+          description: "L'email ou le numéro de téléphone que vous avez entré ne correspond pas avec l'application de demande de nom."
```

Comment:
Here's "application" again ...?

Reply:
I'm just waiting on Ethan on this comment and above Sev. He's talking to Janis. Will get back to you ASAP.

Reply:
Wording for this is now:
"L'email ou le numéro de téléphone que vous avez entré ne correspond pas avec l'application pour votre demande de nom."